frontend/packages/shared/src/types/QueryKey.ts

```diff
@@ -45,6 +45,7 @@ export enum QueryKey {
   Widgets = 'Widgets',
   AppConfig = 'AppConfig',
   IsLoggedInWithAnsattporten = 'IsLoggedInWithAnsattporten',
+  MaskinportenScopes = 'MaskinportenScopes',
```

Comment:
I think we should name it `app-scopes` I think the backend endpoint uses `app-scopes`. 😊